NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -1622,6 +1623,10 @@ static void ProcessReceiveQueue(PARANDIS_ADAPTER *pContext,
                 NET_BUFFER_LIST_NEXT_NBL(*indicateTail) = NULL;
                 (*pnPacketsToIndicateLeft)--;
                 (*nIndicate)++;
+
+                //According to the feedback from customer who originally faced packet loss issue, 
+                //64 is enough and no side effect observed.
+                ret = (!ret) ? (pBufferDescriptor->Queue->GetAvailRecvBufferCount() < 64) : ret;
```

Comment:
I suggest to have a registry key to control this number, in this case is some setups will need different configuration it can be controlled without changing the code. If you think that this is not needed, in anyway please use a constant with meaningful name instead of magic number.

Thanks,
Yan.

Reply:
Thanks , I will add a registry value so that users can control this value. And the perf results will be uploaded here soon.

Reply:
Thanks a lot!